libs/model/src/models/topic.ts

```diff
@@ -29,6 +30,7 @@ export type TopicAttributes = {
   // associations
   community?: CommunityAttributes;
   threads?: ThreadAttributes[] | TopicAttributes['id'][];
+  chain_node?: ChainNodeAttributes;
```

Comment:
should this be called chain_node or something more specific around erc20 weights?

Reply:
I think it's fine. Community chain node is for stake, topic chain node is for erc20.